guide/preferences.md

```diff
@@ -40,6 +40,12 @@ depending on the GPU OpenGL abilities.
 the Camera device images. The methods are sorted from the most efficient one to
 the least efficient one.
 
+- The `Main 3D View Anti-Aliasing` option allows you to enable Multisample Anti-Aliasing
```

Comment:
We have to be consistent with the format already used in the documentation.
Personally I think it could be good to used capital letters in the options name, but I agree with Fabien that `scene` have to be lower case.